docs/mirror_configuration.md

```diff
@@ -1,272 +1,408 @@
-# Mirror configuration
+# Mirror Configuration
 
-The mirror configuration settings are in a configuration section of the configuration file
-named **\[mirror\]**.
+The **\[mirror\]** section of the configuration file contains general options for how Bandersnatch should operate. This includes settings like the source repository to mirror, how to store mirrored files, and the kinds of files to include in the mirror.
 
-This section contains settings to specify how the mirroring software should operate.
+## Examples
 
-## directory
+These examples only show `[mirror]` options; a complete configuration may include [mirror filtering plugins][filter-plugins] and/or options for a [storage backend][storage-backends].
 
-The mirror directory setting is a string that specifies the directory to
-store the mirror files.
+### Minimal
 
-The directory used must meet the following requirements:
+The simplest configuration may only set an output folder:
 
-- The filesystem must be case-sensitive filesystem.
-- The filesystem must support large numbers of sub-directories.
-- The filesystem must support large numbers of files (inodes)
+```ini
+[mirror]
+directory = /srv/pypi
+```
+
+This will mirror index files and package release files from pypi.org. Add configuration for [mirror filtering plugins][filter-plugins] to control what packages are mirrored.
+
+### More Options
 
-Example:
+This shows a few of the more common configuration options:
 
 ```ini
 [mirror]
 directory = /srv/pypi
-```
 
-## json
+; parallel downloads, keep low to avoid overwhelming upstream
+workers = 2
+
+; per-request time limit
+timeout = 15
 
-The mirror json setting is a boolean (true/false) setting that indicates that
-the json packaging metadata should be mirrored in addition to the packages.
+; per-coroutine time limit
```

Comment:
Based on the existing docs and `default.conf` I believe it's the maximum runtime for each aiohttp coroutine, but this would be a good one to track down in the source to confirm that and perhaps explain more clearly.

Reply:
Only if you have time, won't block the PR on it.